rpcd/playbooks/roles/rpc_maas/defaults/main.yml

```diff
@@ -491,7 +505,6 @@ maas_rpc_scripts_dir: /opt/rpc-openstack/scripts
 maas_excluded_checks: []
 
 
-#
```

Comment:
It looks like you might have removed this by mistake. The comment style in the rest of the file has an empty line comment before each comment section, as well as one following the comment section.